packaging/windows/dir.proj

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Package" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I think this was left from a previous version on my branch. If it is no longer needed I will delete it